.nvm_packages

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8; mode: conf-unix -*-
+#
+# Developement tools pre-installed in NVM's node installation [1]
+#
+# [1] https://github.com/nvm-sh/nvm#default-global-packages-from-file-while-installing
+
+eslint
```

Comment:
How do you want to use a globally installed eslint?

I really wish to keep the node configuration in one place: 
* `https://github.com/searxng/searxng/tree/master/searx/static/themes/oscar`
* `https://github.com/searxng/searxng/tree/master/searx/static/themes/simple`

In addition what if system wide node is available?

Reply:
> I really wish to keep the node configuration in one place:

This eslint is only used in development tools like [vscode](https://marketplace.visualstudio.com/items?itemName=abumalick.vscode-nvm) or [emacs](https://github.com/rejeep/nvm.el), therefore the `.nvmrc` is needed, it does not in interfere with the eslint that is installed in the simple theme.

The idea is to have a nodejs version and development tools are installed in the project (NVM).  If you work in different projects each project can have its own environment.  It is similar to the `requirements-dev.txt`. Except that one python interpreter is installed in the OS, what is a drawback of python.

> In addition what if system wide node is available?

It does not interfere with the one from NVM / when the NVM environment is sourced, the PATH variable is modified and the NVM is in front of the PATH variable from the system.

```
$ echo $PATH
/share/searxng/.nvm/versions/node/v16.13.0/bin:/home/markus/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin ...
```